modules/nvidia-gpu-azure-adding-a-gpu-node.adoc

```diff
@@ -0,0 +1,430 @@
+// Module included in the following assemblies:
+//
+//  * machine_management/creating-machinesets/creating-machineset-azure.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-aws-adding-a-gpu-node_{context}"]
+= Adding a GPU node to an existing {product-title} cluster
+
+You can copy and modify a default compute machine set configuration to create a GPU-enabled machine set and machines for the Azure cloud provider.
+
+The following table lists the validated instance types:
```

Comment:
According to [NVIDIA GPU Operator | Platform Support](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/platform-support.html), the following instance types are supported: NC-T4-v3, NC-v3, ND-A100-v4.

I'd say that the [ND-100-V4](https://learn.microsoft.com/en-us/azure/virtual-machines/nda100-v4-series) is missing, no?

Reply:
@fabiendupont QE has not tested with ND-100-V4, if your team has tested that instance on Azure, let's add it to the table

Reply:
I did test it myself :)